generatorcli.py

```diff
@@ -135,6 +135,11 @@ def find_edge_by_nodes(_node_a, _node_b):
             input_file_output.write(f"{command}\n")
     if command == "generate":
         generator = Generator()
-        generator.generate(nodes, edges, signals, filename)
+        config = Config(
+            author_name="Arne Boockmeier",
```

Comment:
This should be either set by some command or should be asked during the CLI process. In the end, its either `Boockmeyer` or some `Default Name`